docs/user-guide/install-ha-sysplex.md

```diff
@@ -0,0 +1,159 @@
+
+   While the procedure to obtain and install the convenience build or SMP/E build are different, the procedure to configure a Zowe runtime are the same irrespective of how the build is obtained and installed.
+
+1. Obtain and install the Zowe build.
+
+   - For how to obtain the convenience build and install it, see [Installing Zowe runtime from a convenience build](install-zowe-zos-convenience-build.md).
+   - For how to obtain the SMP/E build and install it, see [Installing Zowe SMP/E](install-zowe-smpe.md).
+
+**Note:** To allow all LPARs in a Sysplex to access the installation and configuration of Zowe high availability instances, you must install and configure Zowe in a shared file system (zFS directory).
+
+After successful installation of either a convenience build or an SMP/E build, there will be a shared zFS directory that contains the unconfigured Zowe runtime `<RUNTIME_DIR>`, a SAMPLIB library `SZWESAMP` that contains sample members, and a load library `SZWEAUTH` that contains load modules.
+
+## Stage 3: Configure the Zowe high availability runtime
+
+You can configure the Zowe high availability runtime by using JCL and shell scripts. 
+
+**Tip:** It's recommended that you open the links in the following configuration procedure in new tabs.
+
+1. Configure the z/OS security manager to prepare for launching the Zowe started tasks. For instructions, see [Configuring the z/OS system for Zowe](configure-zos-system.md).
+   
+   A SAMPLIB JCL member `ZWESECUR` is provided to assist with the configuration. You can submit the `ZWESECUR` JCL member as-is or customize it depending on site preferences.  
+   
+   If you already have this security step configured from a previous release of Version 1.8 or later, you only need to define Zowe launcher started task security configuration with the following commands.
+
+   - If you use RACF, issue the following commands:
+
+   ```
+   RDEFINE STARTED &ZLNCHSTC..*
+      STDATA(USER(&ZOWEUSER.) GROUP(&STCGRP.) TRUSTED(NO))
+      DATA('ZOWE LAUNCHER')
+
+   SETROPTS RACLIST(STARTED) REFRESH
+   ```
+
+   - If you use CA ACF2, issue the following commands:
+
+   ```
+   SET CONTROL(GSO)
+   INSERT STC.&ZLNCHSTC. LOGONID(&ZOWEUSER.) +
+   GROUP(&STCGRP.) +
+   STCID(&ZLNCHSTC.)
+
+   F ACF2,REFRESH(STC)  
+   ```
+
+   - If you use CA Top Secret, issue the following commands:
+
+   ```
+   TSS ADD(STC) PROCNAME(&ZLNCHSTC.) ACID(&ZOWEUSER.)
+  
+   TSS ADD(&ZOWEUSER.) FAC(STC)   
+
+   ```
+
+   Where, 
+   - ZLNCHSTC is the Zowe launcher task name. Default should be ZWESLSTC.
+   - STCGRP is the group for Zowe started tasks. Default should be ZWEADMIN.
+   - ZOWEUSER is the user ID for the Zowe started task. Default should be ZWESVUSR.
+
+2. Create a VSAM data set which is used by the Caching Service feature of API Mediation Layer. For instructions, see []
```

Comment:
Hi @NayerNajafi , is there a specific topic where people could learn how to create this VSAM data set? We should also add a link for the VSAM step in the diagram so wondering which topic we should link to. 
The APIML has an open PR https://github.com/zowe/docs-site/pull/1506/files. Does this contain any information that we could refer to? Thanks!